exercises/calculator.cpp

```diff
@@ -10,3 +10,37 @@
   Multiplication: 8
   Division: 2
 */
+
+#include <iostream>
+
+using namespace std;
+
+void print_operations(int a, int b)
+{
+  cout << "Sum: " << a + b << endl;
+  cout << "Difference: " << a - b << endl;
+  cout << "Multiplication: " << a * b << endl;
+  cout << "Division: ";
+  if (b == 0)
+  {
+    cout << "impossible" << endl;
+  }
+  else
+  {
+    cout << a / b << endl;
+  }
+}
```

Comment:
potresti scrivere il ritorno a capo fuori da if ed else una volta sola
```c++
cout << endl;
```

Reply:
oppure, potresti....
```C++
  cout << "Division: " << b == 0 ? : "impossibile" :  a / b << endl;
```

per il resto, mi sembra ok 👍

Reply:
Non volevo risultare troppo conciso, ma forse è la cosa migliore

Reply:
> oppure, potresti....
> 
> ```c++
>   cout << "Division: " << b == 0 ? : "impossibile" :  a / b << endl;
> ```
> 
> per il resto, mi sembra ok 👍

L'unico problema è che il compilatore mi da un errore nell'operatore ternario, in quanto le opzioni hanno tipi diversi (const char * e int)

Reply:
fai un cast in char* di a/b :S

Reply:
Con il cast risolvo, anche se al momento dell'esecuzione non riesce a stampare a video il risultato della divisione. Credo sia perché casto un int (di 4 byte) in un char * (di 8 byte)

Reply:
forse andrebbe convertito in altro modo 🤔

Reply:
Cerco un modo alternativo allora